### PR TITLE
Added May events

### PR DESCRIPTION
**Pull request by dasanchez**
Merged the most recent hack night + monthly check-in events into one.

### CODE REVIEW

File: _posts/2017-05-18-install-party.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: "Install Party + Social"
+slug: install-party
+text: 'tomesh monthly install party and social'
+location: Semaphore Studio 307, Claude T. Bissell, 140 St. George St  
+locationLink: http://osm.org/go/ZX6Bw~WNh--?m=
+date: 2017-05-18
```
Comment (benhylau) [inline _posts/2017-05-18-install-party.md]: Are we not doing one this Thursday on the 4th?

Comment (dasanchez) [reply _posts/2017-05-18-install-party.md]: Right! We agreed on a monthly install party and a biweekly hardware (node.tech) meeting at the last planning session-
I understand the one on the 4th is more of a hardware meeting, so should I add it as such? I figure the calendar would end up looking something like:
- May 04 - node.tech
- May 18 - install party/check-in
- Jun 01 - node.tech
- Jun 15 - node.tech
- Jun 29 - install party/check-in
Thoughts?

Comment (benhylau) [reply _posts/2017-05-18-install-party.md]: Oh I thought it was:

May 04 - install party/check-in
May 18 - virtual
Jun 01 - install party/check-in
Jun 15 - virtual

See: https://github.com/tomeshnet/documents/commit/ba2938db12cc25257f3f0ee685d7f3087f261490#diff-b127bd3696cae846707a960e8bed6a7cR80

node.tech are also bi-weekly, but separate from the above, and the first one will be on May 11:

See: https://github.com/tomeshnet/documents/commit/ba2938db12cc25257f3f0ee685d7f3087f261490#diff-b127bd3696cae846707a960e8bed6a7cR190

**Comment by dasanchez** [comment]
I adjusted the dates for all three events coming up this month:
- Monthly install party
- Biweekly node.tech meeting
- Biweekly virtual mesh session

File: _posts/2017-05-18-install-party.md (hunk 1 above)
Comment (dasanchez) [reply _posts/2017-05-18-install-party.md]: Sorry for the confusion! I made a new commit

**Comment by dcwalk** [comment]
We might want to ignore those flakey links (https://github.com/gjtorikian/html-proofer#ignoring-content) for future PRs? I can submit as a separate one

**Comment by dcwalk** [comment]
Also -- just to clarify @benhylau all of the in person events are happening at Semaphore studio?

File: _posts/2017-06-01-install-party.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: "Install Party + Social"
+slug: install-party
+text: 'tomesh monthly install party and social'
+location: Semaphore Studio 307, Claude T. Bissell, 140 St. George St  
+locationLink: http://osm.org/go/ZX6Bw~WNh--?m=
+date: 2017-06-01
```
Comment (dcwalk) [inline _posts/2017-06-01-install-party.md]: I'd suggest bumping this back a week...? and having the June events follow suit

File: _posts/2017-05-04-install-party.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: "Install Party + Social"
+slug: install-party
+text: 'tomesh monthly install party and social'
```
Comment (dcwalk) [inline _posts/2017-05-04-install-party.md]: This shows up in the description so maybe something a little differentiated from the title?

"Bring your equipment to set up a mesh node" (or something?)

File: _posts/2017-05-04-install-party.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+location: Semaphore Studio 307, Claude T. Bissell, 140 St. George St  
+locationLink: http://osm.org/go/ZX6Bw~WNh--?m=
+date: 2017-05-04
+startTime: '19:00'
+endTime: '20:30'
+---
+
+Join us at {{page.startTime | date:"%-I:%M%P "}} on {{ page.date | date: '%B %d, %Y' }} for our monthly install party and working groups' check-in. This month we will be meeting at [{{ page.location }}]({{ page.locationLink }}) for a hands-on session before heading to a nearby bar for drinks.
```
Comment (dcwalk) [inline _posts/2017-05-04-install-party.md]: I'd remove the "working groups'" part? As I think this is more about installing stuff?

**Review by dcwalk — COMMENTED** [review]
Some minor suggestions

File: _posts/2017-05-11-node-tech.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
+---
+layout: post
+title: node.tech Meeting
+slug: node-tech-night
```
Comment (dcwalk) [inline _posts/2017-05-11-node-tech.md]: Maybe drop 'night' from the slug? might be confusing to have a different word.

File: _posts/2017-05-04-install-party.md (hunk 4 above)
Comment (benhylau) [reply _posts/2017-05-04-install-party.md]: But we said we'd do working group check-in during drinks. Maybe:

>Join us at {{page.startTime | date:"%-I:%M%P "}} on {{ page.date | date: '%B %d, %Y' }} for our monthly install party and play with distributed applications. This month we will be meeting at [{{ page.location }}]({{ page.locationLink }}) for a hands-on session before heading to a nearby bar for drinks and progress check with each working group.

File: _posts/2017-05-04-install-party.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: "Install Party + Social"
+slug: install-party
+text: 'tomesh monthly install party and social'
+location: Semaphore Studio 307, Claude T. Bissell, 140 St. George St  
+locationLink: http://osm.org/go/ZX6Bw~WNh--?m=
+date: 2017-05-04
+startTime: '19:00'
+endTime: '20:30'
```
Comment (benhylau) [inline _posts/2017-05-04-install-party.md]: More like 23:00 if we do the social.

File: _posts/2017-05-11-node-tech.md (hunk 5 above)
Comment (benhylau) [reply _posts/2017-05-11-node-tech.md]: We should just drop all the `node.tech` meetings from calendar.

File: _posts/2017-05-25-node-tech.md [hunk 7]
```diff
@@ -0,0 +1,14 @@
+---
```
Comment (benhylau) [inline _posts/2017-05-25-node-tech.md]: Drop.

File: _posts/2017-05-18-virtual-mesh.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+date: 2017-05-18
+startTime: '19:00'
+endTime: '23:00'
+---
+
+Some Toronto Mesh members are connected through a virtual mesh network. This is a transitional phase where our mesh nodes are connected to one another using an internet connection instead of a physical line or radio. We are nonetheless able to test out distributed applications as if we are connected through a physical mesh.
+
+Every other Thursday from **7:00 pm ET** on, we meet virtually on [#virtualmesh:tomesh.net](https://chat.tomesh.net/#/room/#virtualmesh:tomesh.net) and do some hands on hacking. This includes running services on our Raspberry Pi platform and testing distributed applications like [IPFS](https://ipfs.io/). If you would like to join, just hop on our chat and say hi!
+
+Although the [cjdns mesh software](https://github.com/cjdelisle/cjdns) runs on multiple platforms, most people are [running it on a Raspberry Pi](https://github.com/tomeshnet/prototype-cjdns-pi2). No matter how you want to join, we will be happy to help you get meshed up.
```
Comment (benhylau) [inline _posts/2017-05-18-virtual-mesh.md]: https://github.com/tomeshnet/prototype-cjdns-pi now

File: _posts/2017-06-01-install-party.md (hunk 2 above)
Comment (benhylau) [reply _posts/2017-06-01-install-party.md]: The date is correct I think? 2 weeks from last virtual mesh.

**Review by benhylau — CHANGES_REQUESTED** [review]

File: _posts/2017-06-01-install-party.md (hunk 2 above)
Comment (dcwalk) [reply _posts/2017-06-01-install-party.md]: June is a long month. We periodically reset/added a week to keep us in the same week-of-month over the last year

File: _posts/2017-05-11-node-tech.md (hunk 5 above)
Comment (dcwalk) [reply _posts/2017-05-11-node-tech.md]: If you know if it a standing meeting and it is open for people to join, why wouldn't you have them in?

File: _posts/2017-05-04-install-party.md (hunk 3 above)
Comment (dasanchez) [reply _posts/2017-05-04-install-party.md]: Changed

File: _posts/2017-05-04-install-party.md (hunk 4 above)
Comment (dasanchez) [reply _posts/2017-05-04-install-party.md]: Changed to benhylau's text

File: _posts/2017-05-11-node-tech.md (hunk 5 above)
Comment (dasanchez) [reply _posts/2017-05-11-node-tech.md]: I just took them out for now

File: _posts/2017-05-04-install-party.md (hunk 6 above)
Comment (dasanchez) [reply _posts/2017-05-04-install-party.md]: 23:00, done

File: _posts/2017-05-25-node-tech.md (hunk 7 above)
Comment (dasanchez) [reply _posts/2017-05-25-node-tech.md]: Gone

File: _posts/2017-05-18-virtual-mesh.md (hunk 8 above)
Comment (dasanchez) [reply _posts/2017-05-18-virtual-mesh.md]: Updated

**Review by benhylau — APPROVED** [review]

**Pull request merged**